content/docs/user-guide/basic-concepts/dvc-run.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 'dvc run'
```

Comment:
`dvc run` is not a basic concept nor does it need a glossary entry. All cmd names are already linked automatically to their refs.

Original ticket: **glossary: create "stage" entry** #1579